terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,10 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+
+# service configs
+use_set_environment_files = true
+log_level = "trace"
+
+required_cpus = 512
+required_memory = 1024
```

Comment:
Can we use the required_cpus and required_memory refer from the variables.tf as 512 & 1024 ?